generators/server/templates/build.gradle.ejs

```diff
@@ -515,21 +510,3 @@ if (project.hasProperty('nodeInstall')) {
     }
 }
 <%_ } _%>
-
-task renameWarFile (type: Copy) {
-    from ("$buildDir/libs/")
-    include "$project.name-$version-original.war"
-    destinationDir file("$buildDir/libs/")
-    rename "$project.name-$version-original.war", "$project.name-{$version}.war.original"
-}
-
-task deleteWarFile(type: Delete) {
-  delete "$buildDir/libs/$project.name-$version-original.war"
-}
-
-renameWarFile.dependsOn war
-bootWar.dependsOn war,renameWarFile,deleteWarFile
```

Comment:
Good point to have `bootWar` not to depend on the normal `war` tasks as most people do not need the classical `.war` file

Reply:
Yes, they can be generated separately as per [docs](https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/html/#packaging-executable-and-normal)